org.eclipse.xtext.ui.tests/src-longrunning/org/eclipse/xtext/ui/tests/refactoring/ProgressReportingTest.java

```diff
@@ -91,7 +97,55 @@ public void testProgressReportOfRenameCommonDir() throws Exception {
 		Assert.assertFalse(x.exists());
 		Assert.assertFalse(y.exists());
 		List<String> expectation = new ArrayList<>();
-		if (Platform.getBundle("org.eclipse.ltk.core.refactoring").getVersion().compareTo(new Version("3.14.200")) >= 0) {
+		Version ltkVersion = Platform.getBundle("org.eclipse.ltk.core.refactoring").getVersion();
+		if (ltkVersion.compareTo(new Version("3.14.300")) >= 0) {
+			String expected = "BeginTask (1000)\n"
+					+ "Worked 90 (90.0)\n"
+					+ "Worked 505 (595.0)\n"
+					+ "Worked 40 (635.0)\n"
+					+ "SetTaskName Preparing and applying file changes...\n"
+					+ "Worked 19 (654.0)\n"
+					+ "Worked 19 (673.0)\n"
+					+ "Worked 9 (682.0)\n"
+					+ "Worked 10 (692.0)\n"
+					+ "Worked 9 (701.0)\n"
+					+ "Worked 10 (711.0)\n"
+					+ "SetTaskName Creating text changes...\n"
+					+ "Worked 47 (758.0)\n"
+					+ "Worked 20 (778.0)\n"
+					+ "Worked 19 (797.0)\n"
+					+ "Worked 203 (1000.0)\n"
+					+ "BeginTask Creating workspace modifications... (1000)\n"
+					+ "Worked 333 (1333.0)\n"
+					+ "Worked 333 (1666.0)\n"
+					+ "Worked 334 (2000.0)\n"
+					+ "Done\n"
+					+ "BeginTask Performing changes... (1000)\n"
+					+ "Worked 41 (2041.0)\n"
+					+ "Worked 42 (2083.0)\n"
+					+ "Worked 20 (2103.0)\n"
+					+ "Worked 21 (2124.0)\n"
+					+ "Worked 21 (2145.0)\n"
+					+ "Worked 21 (2166.0)\n"
+					+ "Worked 84 (2250.0)\n"
```

Comment:
Oh gosh this progresss reporting is awful. 
41, 20, 84, 41, 41, 125 ...

Anyhow, not our fault.

Reply:
Indeed it is! :'(
At least, by directly comparing strings instead of creating lists of strings, from now on it should be easier to fix it